### DISCUSSION TITLE
Welcome to mcsrvstats Discussions!

### DISCUSSION THREAD

**Discussion by Darkflame72** [AjayACST/mcsrvstats#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sauramel** [comment]
Discord invite no longer valid

**Reply by Darkflame72** [reply]
Yep the discord was shutdown as the main discord bot associated with the project shutdown. Feel free to raise an issue here on github or use the discussions to ask a question.
